tests/test_oom_logger.py

```diff
@@ -193,6 +241,21 @@ def log_line():
     )
 
 
+@pytest.fixture
+def log_line_containerd():
+    return LogLine(
+        timestamp=1720128512,
+        hostname="dev208-uswest1adevc",
```

Comment:
just to make sure that there's nothing in this test that depends on where we run it - can we re-use the hostname from previous tests? i.e.,, 
```suggestion
        hostname="dev37-devc",
```